### PR TITLE
feat(odoo): get custom records

### PR DESCRIPTION
**Pull request by ikus060**
## What does this PR do?

Add a new action to Odoo pieces to query records of any types with any criteria.

ANNOUNCEMENT=true

### CODE REVIEW

File: packages/pieces/community/framework/src/lib/validators/validators.ts [hunk 1]
```diff
@@ -258,7 +258,7 @@ class Validators {
   static integer: TypedValidatorFn<ValidationInputType.NUMBER> = {
     type: ValidationInputType.NUMBER,
     fn: (property, processedValue, userInput) => {
-      if (isInteger(processedValue)) {
+      if (!isInteger(processedValue)) {
```
Comment (abuaboud) [inline packages/pieces/community/framework/src/lib/validators/validators.ts:261]: Ouch good catch, can you bump the package.json for framework.

Comment (ikus060) [reply packages/pieces/community/framework/src/lib/validators/validators.ts:261]: hello  @abuaboud  I will make the change to package.json.

Sry I forgot.

**Comment by ikus060** [comment]
Thanks for your review.

**Comment by ikus060** [comment]
I forget to also pump the version of "framework". Now it's fixed

**Pull request merged**